java/lang/file_permission_others.yml

```diff
@@ -0,0 +1,43 @@
+patterns:
+  - pattern: |
+      {
+        $<!>$<PERMISSIONS:identifier>.add($<OTHER_PERMISSION>);
```

Comment:
Does it not work without the `:identifier` part?

Reply:
Whoops, yes it does!